src/StartPage.elm

```diff
@@ -0,0 +1,72 @@
+module StartPage exposing (..)
+
+import Browser
+import Html exposing (Attribute, Html, button, div, input, pre, text)
+import Html.Attributes exposing (..)
+import Html.Events exposing (onClick, onInput)
+
+
+main =
+    Browser.sandbox { init = init, update = update, view = view }
+
+
+type alias Model =
+    { name : String
+    , team : String
+    , matchNumber : String
+    , driverPos : String
+    , text : String
+    }
+
+
+init : Model
+init =
+    { name = "", team = "", matchNumber = "", driverPos = "", text = "" }
+
+
+type Msg
+    = Name String
+    | Team String
+    | MatchNumber String
+    | DriverStation String
+    | Text String
+
+
+update : Msg -> Model -> Model
+update msg model =
+    case msg of
+        Name newName ->
+            { model | name = newName }
+
+        Team newTeam ->
+            { model | team = newTeam }
+
+        MatchNumber newStation ->
+            { model | matchNumber = newStation }
+
+        DriverStation newStation ->
+            { model | driverPos = newStation }
+
+        Text newText ->
+            if model.name == "" || model.team == "" || model.matchNumber == "" || model.driverPos == "" then
```

Comment:
You repeat `== "" ||` here a lot.
Can you think of a better way to check this (one without duplication)?